File: Ej39/Java/TercerEntero.java
import java.util.Scanner;

public class TercerEntero {
	private static final int MAXIMO = 100;
	public static void main(String[] args){
		int i;
		int suma=0;
		for (i=2; i < MAXIMO; i+=3) {
			suma+=i;
		}
		System.out.println("\tSuma de cada tercer entero desde el dos");
		System.out.println("Suma: " + suma);
	}
}

File: Ej39/C++/tercerEntero.c
#include <stdio.h>

#define MAXIMO 100

int main(){
	int i;
	int suma=0;
	for (i=2; i < MAXIMO; i+=3) {
		suma+=i;
	}
	printf("\tSuma de cada tercer entero desde el dos\n");
	printf("Suma: %i\n", suma);
	return 0;
}